portfolio/src/main/java/com/google/sps/data/BookDaoDatastore.java

```diff
@@ -148,7 +149,11 @@ private Entity parseEntity(Book book) {
         bookEntity.setProperty(CATEGORIES,book.categories());
         bookEntity.setProperty(AUTHORS,book.authors());
         bookEntity.setProperty(LANGUAGE,book.language());
-        bookEntity.setProperty(DESCRIPTION,book.description());
+        if (book.description().getBytes().length >= 1500) {
+            bookEntity.setProperty(DESCRIPTION, new Text(book.description()));
```

Comment:
Is there a reason we don't just always do "new Text(book.description())" all the time? Like why only do it when the length is >1500?

Reply:
So far I  only changed it to a text object when required. I can change it so that they are all Text objects, I don't think it should cause any issues.